packages/server/src/service/trpc.ts

```diff
@@ -109,12 +109,13 @@ export class VbmRpc implements ImplRpc<IVbmRpc, Request> {
     const method = toContactMethod(info.contact)
     if (!method) return error('Unable to find contact details for local official')
 
-    // Adds timestamp to new sign ups
+    // Adds timestamp to new sign ups, these are used to help cronjob queries
```

Comment:
This extra comment seems unnecessary -- there are a million reasons to have a timestamp for signups.